src/grpc/tests/function/echo_async_client.cpp

```diff
@@ -372,7 +405,6 @@ class TestServer {
         static void set_response(BufT const& req, grpc::ByteBuffer& resp, bool set_buf) {
             DataMessage cli_request;
             DeserializeFromBuffer(req, cli_request);
-            RELEASE_ASSERT((cli_request.m_buf == GENERIC_CLIENT_MESSAGE), "Could not parse response buffer");
```

Comment:
Is it not possible to check for substr of Generic_client_message? I am not sure there is another place where we check the consistency of the message after send.

Reply:
yes it is checked here
https://github.com/eBay/sisl/blob/dfacbfda39b192209ef13af452d11dde7e4c17e0/src/grpc/tests/function/echo_async_client.cpp#L170